prospero-cli/src/test/java/org/wildfly/prospero/cli/commands/InstallCommandTest.java

```diff
@@ -107,20 +111,20 @@ public void errorIfChannelsIsNotPresentAndUsingCustomFplOnInstall() {
                 CliConstants.FPL, "foo:bar");
         assertEquals(ReturnCodes.INVALID_ARGUMENTS, exitCode);
         assertTrue("output: " + getErrorOutput(), getErrorOutput().contains(String.format(
-                CliMessages.MESSAGES.prosperoConfigMandatoryWhenCustomFpl().getMessage(), CliConstants.PROVISION_CONFIG)));
+                CliMessages.MESSAGES.channelsMandatoryWhenCustomFpl().getMessage(), CliConstants.CHANNELS)));
```

Comment:
String.format is not needed, the new message doesn't take a parameter

Reply:
You are right!